### DISCUSSION TITLE
VXLAN on NSX?

### DISCUSSION THREAD

**Discussion by sei-avershave** [rancher/rke2#6059] [Q&A]
Just a quick question! If I'm installing RKE2 do I have to change the backend port to 4789 if I'm using NSX? I ran into issues in the past with running RKE2 with VXLAN on NSX but I couldn't figure out what was happening but I believe this could be the issue.

Thanks!

**Comment by manuelbuil** [comment]
I'm not familiar with NSX but it depends on what NSX is expecting as VNI, doesn't it state it in the requirements?

**Reply by sei-avershave** [reply]
Sure but before sending the packet containing the VNI in the payload, does the backend port have to match the NSX VXLAN port?

**Reply by brandond** [reply]
Are you trying to use the Cisco ACI CNI Plugin, or are you just using another CNI on top of NSX? If you're not integrating into NSX I don't see why it would care what port you're using. If you ARE using the ACI CNI plugin then their documentation should cover the requirements.

**Reply by sei-avershave** [reply]
Gotcha! I was just curious because as it was stated, I was just using it out of the box and it wasn't working. Is there conflict running the default CNI regarding VXLAN because it did not work. I might also be posting in the wrong discussion board and might move over to flannel.

**Reply by manuelbuil** [reply]
I think I understand you now. Flannel assumes that all vteps use the same port, so it hardcodes the dstport to the set port, i.e. there is no mechanism that detects what the destination vtep is using as port
